README.md

```diff
@@ -1,90 +1,43 @@
+- **[Sector](https://github.com/meraxes-devs/sector):** A library for computing spectral energy distributions from Meraxes simulations.  
 
 
-Attribution
------------
+## **Installation**
+Please refer to [`BUILD.md`](./BUILD.md) for detailed installation and build instructions.
 
-If you use this code or its results in any publications, please cite [Mutch et al. 2016b](http://adsabs.harvard.edu/abs/2016MNRAS.462..250M) (MNRAS, 462, 250):
 
-```
-@ARTICLE{2016MNRAS.462..250M,
-   author = {{Mutch}, S.~J. and {Geil}, P.~M. and {Poole}, G.~B. and {Angel}, P.~W. and 
-	{Duffy}, A.~R. and {Mesinger}, A. and {Wyithe}, J.~S.~B.},
-    title = "{Dark-ages reionization and galaxy formation simulation - III. Modelling galaxy formation and the epoch of reionization}",
-  journal = {\mnras},
-archivePrefix = "arXiv",
-   eprint = {1512.00562},
- keywords = {galaxies: formation, galaxies: high redshift, dark ages, reionization, first stars},
-     year = 2016,
-    month = oct,
-   volume = 462,
-    pages = {250-276},
-      doi = {10.1093/mnras/stw1506},
-   adsurl = {http://adsabs.harvard.edu/abs/2016MNRAS.462..250M},
-  adsnote = {Provided by the SAO/NASA Astrophysics Data System}
-}
-```
+## **Documentation**
+Comprehensive documentation is currently under development. For inquiries or assistance, please contact the team.  
 
-If you use any of the following features, please also cite the relevant papers:
 
-AGN model
-: [Qin et al. 2017 (arXiv:1703.04895)](https://ui.adsabs.harvard.edu/#abs/2017arXiv170304895Q/abstract)
+## **Acknowledging**
+If you use Meraxes or its outputs in your research, please cite the following foundational paper:  
 
-Minihalo model
-: [Ventura et al. 2024 (arXiv:2401.07396)](https://ui.adsabs.harvard.edu/abs/2024MNRAS.529..628V/abstract)
+- Mutch et al. (2016). **Dark-ages reionization and galaxy formation simulation - III. Modelling galaxy formation and the epoch of reionization.**  *Monthly Notices of the Royal Astronomical Society*, 462 (1): 250–276. [DOI: 10.1093/mnras/stw1506](https://doi.org/10.1093/mnras/stw1506).  
 
-Contributors
-------------
+If using specific features introduced in Meraxes, please cite the corresponding papers:  
 
-* **Simon Mutch** (Lead author; *The University of Melbourne*)
-* Paul Geil (*The University of Melbourne*)
-* Yuxiang Qin (*The University of Melbourne*)
-* Hansik Kim (*The University of Melbourne*)
-* Greg Poole (*Astronomy Data and Compute Services*)
-* Yisheng Qiu (*The University of Melbourne*)
-* Brad Greig (*The University of Melbourne*)
-* Emanuele Maria Ventura (*The University of Melbourne*)
+- **AGN Model:**  Qin et al. (2017). **Dark-ages reionization and galaxy formation simulation - X. The small contribution of quasars to reionization.** *Monthly Notices of the Royal Astronomical Society*, 472 (2): 2009–2027. [DOI: 10.1093/mnras/stx1909](https://doi.org/10.1093/mnras/stx1909).  
 
```

Comment:
Maybe adding Qiu+19 for the supernova model? (As you prefer though)

Reply:
will do